evennia/prototypes/protfuncs.py

```diff
@@ -325,3 +326,15 @@ def objlist(*args, **kwargs):
 
     """
     return ["#{}".format(obj.id) for obj in _obj_search(return_list=True, *args, **kwargs)]
+
+
+def dbref(*args, **kwargs):
+    """
+    Usage $dbref(<#dbref>)
+    Returns one Object searched globally by #dbref. Error if #dbref is invalid.
+    """
+    _RE_DBREF = re.compile(r"\#[0-9]+")
```

Comment:
This can just as well go into the outer scope of this module so it's only compiled once.